components/keysender/actions/create-transaction/create-transaction.mjs

```diff
@@ -0,0 +1,65 @@
+import keysender from "../../keysender.app.mjs";
+import constants from "../../common/constants.mjs";
+
+export default {
+  key: "keysender-create-transaction",
+  name: "Create Transaction",
+  description: "Creates a new transaction within Keysender. [See the documentation](https://panel.keysender.co.uk/api#tag/Transaction/paths/~1transaction~1addcustom/post)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    keysender,
+    payer: {
+      type: "string",
+      label: "Payer",
+      description: "Email address of the payer",
+    },
+    databaseId: {
+      propDefinition: [
+        keysender,
+        "databaseId",
+      ],
+    },
+    amount: {
+      type: "string",
```

Comment:
Is this supposed to be a string? API documentation lists it as a float, so maybe it should be a string prop but converted to a number (and checked to be a valid number) before being sent